tests/go/GofmtBearTest.py

```diff
@@ -1,7 +1,15 @@
 from bears.go.GofmtBear import GofmtBear
 from coalib.testing.LocalBearTestHelper import verify_local_bear
 
-GofmtBear = verify_local_bear(
+GofmtBearTest = verify_local_bear(
     GofmtBear,
     ('package main\n\nfunc main() {\n\treturn 1\n}',),
     ('package main\nfunc main() {\n\treturn 1\n}',))
+
+GofmtBearWithSettingsTest = verify_local_bear(
```

Comment:
Now the comment I was going to make originally, but avoided because I was too busy scratching my head, is that this doesnt really test very much.  It only confirms that using setting `simplify` doesnt break the bear.

It doesnt confirm that the bear autoformats the bear.  To do that, you should use `coalib.testing.LocalBearTestHelper`'s `check_result` (and maybe even build a better check method in `LocalBearTestHelper` that verifies input A is converted to output A')